common/persistence/nosql/nosqlplugin/cassandra/gocql/session.go

```diff
@@ -36,12 +37,18 @@ import (
 	"go.temporal.io/server/common"
 	"go.temporal.io/server/common/log"
 	"go.temporal.io/server/common/log/tag"
+	"go.temporal.io/server/common/metrics"
 )
 
 var _ Session = (*session)(nil)
 
 const (
-	sessionRefreshMinInternal = 5 * time.Second
+	sessionRefreshMinInternalSecs = 5
```

Comment:
nit: why def as a number? time.Duration has a String() method and will print human readable string I think.